assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -62,12 +71,15 @@ const PRESETS = [
       )}</span>`,
       fontWeight: FONT_WEIGHT.BOLD,
       fontSize: dataFontEm(2.667),
-      lineHeight: 1.1,
+      lineHeight: 1.19,
       x: DEFAULT_LEFT_MARGIN,
       y: (PAGE_HEIGHT - dataFontEm(2.667)) / 2,
       font: TEXT_ELEMENT_DEFAULT_FONT,
-      width: DEFAULT_ELEMENT_WIDTH,
+      // Measured in editor. As small as you can make it before line wraps
+      width: 181,
```

Comment:
Thoughts on `DEFAULT_HEADING_WIDTH` and `DEFAULT_HEADING_2_WIDTH` (and so on) to prevent magic numbers?